packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -35,7 +35,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:master-2a236bb79'
+    return 'defi/defichain:master-57c099c1a'
```

Comment:
This docker image is the latest as of this writing. Reviewer, kindly change the value if this is not the latest when you review this PR.